### PR TITLE
amd_define option

### LINKED ISSUE #4: options.variable + options.amd_wrapper
Hi,
If I define a plain variable like `templates` to store the templates in combination with amd_wrapper, this variable is not declared: there is no `var`keyword. This leads to an undefined error in the logical OR.
Plus, wouldn't it be better if the amd wrapper were defined as a module effectively returning something?
Something like this:
```
define(["twig"], function(Twig) {
    var templates = templates || {};
    templates["project.html"] = Twig.twig({...});
    templates["home.html"] = Twig.twig({...});
    return templates; 
});
```

### PR DESCRIPTION
**Pull request by ekyna**
Added amd_define option (+test).
Updated Readme (options.variable default value).
Fix #4 issue (missing var keyword).
Fix tests on Windows (don't normalize linefeeds).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +52,26 @@ Default value: `true`
...
+
+Determines whether to define the AMD module or not.
+#### Examples
+* `options.amd_define: true` 
+    ```javascript 
+    define(["twig"], function(Twig) { ... return windows.JST; });
+    ``` 
+* `options.amd_define: 'my-templates'`
+    ```javascript 
+    define('my-templates', ["twig"], function(Twig) { ... return windows.JST; });
```
Comment (adamdicarlo) [inline README.md]: windows → window

**Comment by adamdicarlo** [comment]
@ekyna Do you have any interest in maintaining this project? I haven't used Grunt or Twig in a long time, so I don't have a lot of motivation (or even expertise) for maintaining this anymore 😞

File: README.md (hunk 1 above)
Comment (adamdicarlo) [inline README.md]: Nevermind, it let me add a commit to fix these!

**Comment by adamdicarlo** [comment]
Either way, thanks for contributing - merged!

**Comment by ekyna** [comment]
I plan to use your lib in some projects, so i'll probably make some PR. But i'm enough skilled to maintain it, sorry :s. Thx for merging <3

**Pull request merged**